VRDR.CLI/Program.cs

```diff
@@ -1101,6 +1104,32 @@ record = coding.DeathRecord;
                 Console.WriteLine(payload);
                 return 0;
             }
+            else if (args.Length == 2 && args[0] == "filter")
+            {
+                Console.WriteLine($"Filtering file {args[1]}");
+
+                BaseMessage baseMessage = BaseMessage.Parse(File.ReadAllText(args[1]));
+                
+                FilterService FilterService = new FilterService("./VRDR.Filter/NCHSIJEFilter.json", "./VRDR.Filter/IJEToFHIRMapping.json");
+
+                string filteredFile;
+                
+                try
+                {
+                    filteredFile = FilterService.filterMessage(baseMessage).ToJson();
+                    BaseMessage.Parse(filteredFile);
+                    Console.WriteLine($"File successfully filtered");
+                    
+                    string filteredFilePath = "./VRDR.Filter/filteredFile.json";
+                    File.WriteAllText(filteredFilePath, filteredFile);
+                }
+                catch (Exception e)
+                {
+                    Console.WriteLine($"Error occured while filtering file ({e})");

```

Comment:
If an error occurs, please just let the exception be thrown so the full stack trace shows, remove the try/catch block.